### PR TITLE
Receipt UI

### PR DESCRIPTION
**Pull request by SidonieBouthors**
- [x] Add basic structure of add receipt page
- [x] Add date picker

### CODE REVIEW

**Comment by SidonieBouthors** [comment]
This is a pretty basic start with a layout and a date picker. I'll be working more on adding the camera and some functionality next sprint but I think we can merge this as a start for now

File: app/src/main/java/com/github/se/assocify/navigation/NavigationActions.kt [hunk 1]
```diff
@@ -24,4 +24,8 @@ class NavigationActions(private val navController: NavHostController) {
   fun onAuthError() {
     /* Handle auth error */
   }
+
+  fun back() {
+    /*TODO handle back navigation*/
```
[the remark this answers is not in the document]
Comment (ortoLover) [reply app/src/main/java/com/github/se/assocify/navigation/NavigationActions.kt:29]: Everything seems to work well. I wrote a preview that seems to work as intended, the application still works as before and the test pass. However all the TODOS make it difficult to understand what was forgetten and what is expected to be finished in the next weeks.

**Review by ortoLover — APPROVED** [review]
The PR seems to work well, the app works as before and the new modifications seem to work as well.

File: app/src/main/java/com/github/se/assocify/navigation/NavigationActions.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (SidonieBouthors) [reply app/src/main/java/com/github/se/assocify/navigation/NavigationActions.kt:29]: Many of them will be fixed next week when I do the functionality of this part and link things together. I'll make sure to double check that I fix all of them in my next PR :)

**Pull request merged**